lib/airtable/entity/record.rb

```diff
@@ -0,0 +1,92 @@
+require 'time'
+# Main object for store Airtable Record entity
+module Airtable
+  module Entity
+    class Record
+      extend Forwardable
+      attr_reader :id, :created_at, :fields
+
+      def_delegators :@fields, :[], :[]=
+
+      def initialize(id, options = {})
+        @id = id
+        parse_options(options)
+      end
+
+      def new_record?
+        @id.nil? || @id.empty?
+      end
+
+      def __create__(base, name)
+        res = base.__make_request__(:post, name, { fields: fields })
```

Comment:
this `name` needs escaping equivalent to JS encodeURIComponent (e.g. https://github.com/Airtable/airtable-ruby/commit/3cc21eaf699be2eec2c0c135c042f61508cff878). I had hard time finding a better way to do it.